scripts/compute_speed_table.py

```
@@ -0,0 +1,635 @@
+    cache = os.path.join(CACHE_DIR, f"{feed.key}.zip")
+    if not os.path.exists(cache):
+        url = feed.url
+        if feed.needs_token:
+            if not token:
+                sys.stderr.write(
+                    f"  {feed.name}: ODPT_ACCESS_TOKEN 未設定のためスキップ"
+                    "(https://developer.odpt.org/ で無料発行できます)\n"
+                )
+                return None
+            url = url.format(token=token)
+        req = urllib.request.Request(url, headers={"User-Agent": USER_AGENT})
```

Ruff (0.15.20)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)